net/strongswan/Makefile

```diff
@@ -109,6 +109,7 @@ include $(INCLUDE_DIR)/nls.mk
 define Package/strongswan/Default
   SUBMENU:=VPN
   SECTION:=net
+  MENU:=1
```

Comment:
This is meant to be in `Package/strongswan`, although I'm not sure it hurts to be here.

Reply:
Done, thanks!